### ISSUE TITLE
Deprecated warning in PHP 8.2.8

### ISSUE THREAD

**Issue by ggieling** [mmucklo/krumo#67]
I get the below warning when using krumo 0.7.0 in a php 8.2.8 environment

Deprecated: Use of "static" in callables is deprecated in ..../class.krumo.php on line 1522
Deprecated: Use of "static" in callables is deprecated in ..../class.krumo.php on line 1523

corresponding lines in class.krumo.php are:
```
1522            $_    = preg_replace_callback( "/^([ \t]+)/", "static::convert_whitespace", $_);
1523            $_    = preg_replace_callback( "/([ \t]+)$/", "static::convert_whitespace", $_);
```

I don't know when this will change from deprecated to forbidden, but it is probably relevant to tackle this before the change.

**Comment by scottchiefbaker** [comment]
I just landed https://github.com/mmucklo/krumo/commit/0052ffd9343073b7a29903b8a7f7efb3473869da which should fix this issue. Thanks for the heads up. I don't have PHP 8.2 in production yet, so I haven't tested much with it.

Can you confirm this fixes the issue for you?

**Comment by driehle** [comment]
@scottchiefbaker While 0052ffd indeed fixes this issue, unfortunately, no release has yet been made with that change.

Would it be possible to make a new release? Remember to update the `Kruma::VERSION` constant before... ;-)

**Comment by scottchiefbaker** [comment]
I just released v0.7.2 with this fix. Give it a try.
